user-interface/src/case-assignment/CaseAssignmentScreen.tsx

```diff
@@ -61,7 +61,7 @@ export const CaseAssignment = () => {
         caseList.forEach((bCase) => {
           bCase.assignments = bCase.assignments ?? [];
         });
-        caseList.sort(sortByDate).sort(sortByCaseId);
+        caseList.sort(sortByCaseId).sort(sortByDateFiled);
         setCaseList(caseList);
```

Comment:
This helped get everything sorted by date filed.... but I don't think this is going to work sorting one way and then the other.  The problem is that the sorts aren't build upon one another.  We're sorting the entire list one way and then sorting it another all over again.

Reply:
I would assume that chaining sort functions like this isn't going to result in what we want.  But I could be wrong...

Reply:
Yep. Array.sort is an in place sort. The sorting comparator has to prioritize the two facets together in the same function.